sway_libs/src/signed_integers/i128.sw

```diff
@@ -0,0 +1,173 @@
+library i128;
+
+use core::num::*;
+use std::assert::assert;
```

Comment:
This is part of the prelude and doesn't need to be included anymore.

Reply:
Removed here, but sometimes it's actually needed